src/components/irrecoverable-login-error.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useService } from './container-context.js';
+import { Trans, useTranslation } from 'react-i18next';
+import ClientConfig from '../bootstrap/client-config.js';
+
+export default function IrrecoverableLoginError({ type }) {
```

Comment:
How about going with`BlockedLoginMessage` instead of `IrrecoverableLoginError`? To me all these cases are actually recoverable since it's not a system failure.

Reply:
That is a much better name, thanks! Will do!

Reply:
Done!